Lesson7/AdditionalTask/Validator.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Reflection;
+
+namespace AdditionalTask
+{
+    public class Validator
+    {
+        public (bool isValid, string name) Validate(Worker worker)
+        {
+            var type = worker.GetType();
+
+            var accessLevel = type.GetCustomAttribute<AccessLevelAttribute>(true).AccessLevel;
```

Comment:
what will happen if `type` does not contain  `AccessLevelAttribute`?

Reply:
All types that are checked are inhereted from abstract class Work that contains AccessLevelAttribute with a property value of 0. If the derived class dosen't have this type of  attribute it will be taken from its based class. It works because we specified true in the method parameters.